src/problem1.js

```diff
@@ -1,6 +1,39 @@
 function problem1(pobi, crong) {
-  var answer;
-  return answer;
+  if (errorCheck(pobi)) return -1;
+  if (errorCheck(crong)) return -1;
+
+  const pobiScore = getMaxNum(pobi);
+  const crongScore = getMaxNum(crong);
+  const result = compareTwoScores(pobiScore, crongScore);
+
+  return result;
+}
+
+function errorCheck(pages) {
+  if (pages[1] - pages[0] !== 1) return true;
+  if (pages[1] > 400) return true;
+  if (pages[0] < 1) return true;
```

Comment:
```suggestion
function errorCheck([left, right]) {
  if (right - left !== 1) return true;
  if (right > 400) return true;
  if (left < 1) return true;
```
이렇게 수정해주면 좋을 것 같아요! 구조분해 할당을 사용하면 배열을 풀어줄 수있어서 가독성이 좋아질 거에요!